core/src/main/java/org/eclipse/hono/util/TenantConstants.java

```diff
@@ -67,7 +67,7 @@
     /**
      * The name of the Tenant API endpoint.
      */
-    public static final String TENANT_ENDPOINT = "tenant";
+    public static final String TENANT_ENDPOINT = "tenants";
```

Comment:
this endpoint is defined by the Tenant API and cannot be changed by the device registry's HTTP management API
please revert

Reply:
Done.